From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Sun, 14 Oct 2012 05:28:59 +0900
Subject: Remove obsolete Encoding Key from desktop files

The Encoding key is now deprecated by the FreeDesktop standard and
all strings are required to be encoded in UTF-8. This desktop entry
explicitly specifies an Encoding of UTF-8, which is harmless but
no longer necessary.

Refer to
  http://standards.freedesktop.org/desktop-entry-spec/1.0/apc.html
for details.

Signed-off-by: Youhei SASAKI <uwabami@gfd-dennou.org>
---
 data/cairo-dock-cairo.desktop |    1 -
 data/cairo-dock.desktop       |    1 -
 2 files changed, 2 deletions(-)

diff --git a/data/cairo-dock-cairo.desktop b/data/cairo-dock-cairo.desktop
index 83ec875..1b9c7a1 100755
--- a/data/cairo-dock-cairo.desktop
+++ b/data/cairo-dock-cairo.desktop
@@ -1,6 +1,5 @@
 
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Exec=cairo-dock -c
 Icon=cairo-dock
diff --git a/data/cairo-dock.desktop b/data/cairo-dock.desktop
index 93f8b46..f009660 100755
--- a/data/cairo-dock.desktop
+++ b/data/cairo-dock.desktop
@@ -1,6 +1,5 @@
 
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Exec=cairo-dock
 Icon=cairo-dock
